cmds/help.js

```diff
@@ -117,20 +119,23 @@ function cmd_help(lang, msg, args, line, wiki) {
 		else msg.reactEmoji('❓');
 	}
 	else if ( msg.isAdmin() && pause[msg.guild.id] ) {
+		var embed = new MessageEmbed().setFooter( 'Help' ).setTimestamp();
 		var cmdlist = lang.get('help.pause') + '\n';
 		cmdlist += formathelp(helplist.pause, msg, lang);
-		msg.sendChannel( cmdlist, {split:{char:'\n🔹',prepend:'🔹',maxLength}}, true );
+		embed.setDescription( cmdlist );
+		msg.replyMsg( '', {embed}, false, false )
 	}
 	else {
-		var cmdlist = lang.get('help.all') + '\n';
+		var embed = new MessageEmbed().setTitle( lang.get('help.all') ).setFooter( 'Help' ).setTimestamp();
 		helplist.default.forEach( cmd => {
 			if ( ( !restrictions.fandom.includes( cmd ) || wiki.isFandom(false) ) && 
 			( !restrictions.minecraft.includes( cmd ) || isMinecraft ) ) {
 				cmdlist += formathelp(helpmap[cmd], msg, lang) + '\n';
 			}
 		} );
 		cmdlist += '\n🔸 ' + lang.get('help.footer');
-		msg.sendChannel( cmdlist, {split:{char:'\n🔹',prepend:'🔹',maxLength}} );
+		embed.setDescription( cmdlist );
+		msg.replyMsg( '', {embed}, false, false )
```

Comment:
This will break for languages where the `cmdlist` is above the maximal character limit as it is no longer split into multiple messages.